index.js

```diff
@@ -184,7 +185,27 @@ function resolveWebpackPath(partial, filename, directory, webpackConfig) {
 
   try {
     var loadedConfig = require(webpackConfig);
-    var resolver = webpackResolve.create.sync(loadedConfig.resolve || {});
+    var resolveConfig = objectAssign({}, loadedConfig.resolve);
+
+    resolveConfig.modules = [];
```

Comment:
Wouldn't this blow away the existing value in `resolve.modules`? Shouldn't it be a default to `[]` if `modules` is empty?